src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,151 @@
+.NavbarItem {
+  background: linear-gradient(90deg, rgb(8, 8, 8) 0%, rgb(77, 77, 77) 100%);
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar-logo {
+  color: #fff;
+  justify-self: start;
+  margin-left: 20px;
+  cursor: default;
+  transition: all 1s ease-in-out;
+}
+
+.nav-menu {
+  text-align: center;
+  list-style: none;
+  display: grid;
+  grid-template-columns: repeat(6, auto);
+  grid-gap: 10px;
+  width: 70vw;
+  justify-content: end;
+  margin-right: 2rem;
+}
+
+.nav-links {
+  color: #fff;
```

Comment:
pomyśl o jakiejś strukturze atrybutów, żeby nie deklarować tego po kilka razy. Do jedna z najlepszych rzeczy w SCSS ;)

.container{
   h1{
      font-size: 25px;
      color:#E45456;
   }
   
   p{
      font-size: 25px;
      color:#3C7949;
   }

   .box{
      h1{
         font-size: 25px;
         color:#E45456;
      }
      
      p{
         font-size: 25px;
         color:#3C7949;
      }
   }
}

Reply:
Na ile sie dało to poprawiłem :D